addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -194,6 +194,14 @@ def _l10n_in_edi_ewaybill_cancel_invoice(self, invoices):
             res[invoices] = inv_res
         return res
 
+    def _l10n_in_edi_ewaybill_handle_zero_distance_alert_if_present(self, invoices, response):
```

Comment:
@hamo-odoo But you can change it hear at least
```suggestion
    def _l10n_in_edi_ewaybill_handle_zero_distance_alert_if_present(self, invoice, response):
```